<launch>
	<node pkg="dataset_player" type="dataset_player_node" name="dataset_player_node" output="screen" launch-prefix="gdb -ex run --args">

		<param name="datasetType" type="string" value="KITTI"/>

		<param name="pathOfDataset" type="string" value="/home/zlf/docker/dataset"/>
		<param name="subDirectory" type="string" value="00"/>

		<param name="pubPointCloud" type="bool" value="true"/>
		<param name="pointCloudTopic" type="string" value="/velodyne_points"/>
		<param name="pointCloudRate" type="double" value="10.0"/>
		<param name="pointCloudFrameID" type="string" value="velodyne"/>

		<param name="pubImageColor" type="bool" value="true"/>
		<param name="pubImageGry" type="bool" value="false"/>
		<param name="isStereo" type="bool" value="false"/>
		<param name="imageTopic" type="string" value="/camera"/>
		<param name="imageRate" type="double" value="10.0"/>
		<param name="imageFrameID" type="string" value="camera"/>

		<param name="pubGroundTruth" type="bool" value="false"/>
		<param name="groundTruthTopic" type="string" value="/imu/data"/>
		<param name="groundTruthRate" type="double" value="100.0"/>
		<param name="groundTruthRate" type="string" value="base_link"/>
	</node>

	<!-- node pkg="rviz" type="rviz" name="rviz" args="-d $(find dataset_player)/rviz/datasetplayer.rviz" required="true" / -->
</launch>